plugins/outputs/http/http_test.go

```diff
@@ -170,11 +170,22 @@ func TestStatusCode(t *testing.T) {
 			plugin: &HTTP{
 				URL: u.String(),
 			},
-			statusCode: http.StatusMultipleChoices,
+			statusCode: http.StatusBadRequest,
```

Comment:
Please don't modify existing tests but rather add another one. This makes sure that previous behavior will not change with the PR...

Reply:
Here I actually meant to fix a misleading test, it should test for 4xx statuscodes but StatusMultipleChoices is statuscode 300. I'm fine with reverting this if you'd rather have this fixed in a separate PR.
(The code currently treats 3xx and 4xx the same but that might change in the future and then a more accurate test would help)

Reply:
I completely missed that fact of the test being "wrong"... Please keep as is.